source/includes/api/tables/progress-response.rst

```diff
@@ -51,13 +51,18 @@
    * - ``collectionCopy``
        ``.estimatedTotalBytes``
      - integer
-     - Estimated total number of bytes to be copied.
+     - Estimated total number of bytes to be copied globally by all ``mongosync``
+       instances.
 
    * - ``collectionCopy``
        ``.estimatedCopiedBytes``
      - integer
      - Estimated number of bytes which have been copied to the
-       destination cluster.
+       destination cluster by this ``mongosync`` instance. 
+       
+       To calculate the total estimated progress, add the value of  the
```

Comment:
```suggestion
       To calculate the total estimated progress as a value from 0.0-1.0, add the value of  the
```

Or you could suggest multiplying by 100 to get a percentage.

Reply:
Fixed